packages/link/src/components/Link.story.js

```diff
@@ -144,4 +144,12 @@ storiesOf("Link/Featured", module)
   .add(
     "no href with left icon",
     withInfo()(() => <Link leftIcon="wg-place">Awesome Link</Link>)
+  )
+  .add(
+    "Custom color",
+    withInfo()(() => (
+      <Link href="#" type="hotpink">
+        Awesome Link
+      </Link>
+    ))
```

Comment:
The featured link starts with the color, so having an undefined hover wont be an issue, will we need type=neutral behavior on liks with other colors? (no underline and a link color on hover)